stats.rst

```diff
@@ -4,14 +4,89 @@
 Stats Quick Start
 =================
 
-A quick guide to getting the internal stats collection working in flux.
-This guide assumes you have already built a flux binary; if you have not see :ref:`quickstart`.
+Flux includes an optional instrumentation facility to collect internal metrics, or "stats",
+and send them to an aggregator for eventual display in a front end graphing tool such as Graphite.
+The internal stats collection in flux is built on the `statsd <https://github.com/statsd/statsd>`_ protocol and
+following the typing convention used by `brubeck <https://github.com/lukepalmer/brubeck>`_.
+Flux stats supports three metric types: counters, gauges, and timings.
 
-.. _brubeck:
+ * Counter - An integer value that will, on the back end (brubeck) send the count and reset to 0 at each flush. It calculates the change from the value sent at the previous flush. An example of where to use a counter is the builtin message counters that are part of each flux_t handle. The counts will continually increase and brubeck will handle calculating the count sent in the interval.
+ * Gauge   - An integer that takes an arbitrary value and maintains its value until it is set again. Gauges can also take incremental values in the form of a + or - in front of the value which increments the previously stored value. An example of where to use a gauge is to track the current size of the broker's content-cache. At each point, the cache's sizes are independent of each other.
+ * Timing  - A double value which represents the time taken for a given task in ms. An example of where to use a timer is timing the length of asynchronous loads in the broker's content-cache. The cache entry can keep track of when the load was started and then calculate and send the time taken once the entry is loaded.
 
-------------------
+For more information on statsd types see the `statsd docs <https://github.com/statsd/statsd/blob/master/docs/metric_types.md>`_.
+
+------------
+Using Docker
+------------
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+Running Brubeck & Graphite
+^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Seems like the major heading here should be "running brubeck and graphite" and the minor ones "using docker" and, later, "using git/package manager" or similar.

Also, at the beginning of the "running brubeck and graphite" section, a short intro might be useful, like:
> Two methods for starting brubeck for stats aggregation and graphite for stats display are presented below.   The docker method is recommended, if docker is available.  Both methods configure brubeck to accept stats messages from Flux on UDP port 8126, and graphite to capture stats from brubeck and present them via a web server  at  `http://127.0.0.1`.